stdlib/merkleTree/sha256PathProof.code

```diff
@@ -0,0 +1,31 @@
+import "../hashes/sha256/512bit.code" as sha256
+import "../utils/multiplexer/256bit.code" as multiplex
+import "../utils/binary/not.code" as NOT
+
+// Merke-Tree inclusion proof for tree depth 3 
```

Comment:
As far as I understand, we can't make this generic over `treeDepth` yet so maybe the file should be named accordingly (for depth 3 here) `sha256PathProof3` or something